packages/app/src/components-styled/sidebar-metric/sidebar-metric.tsx

```diff
@@ -28,7 +27,7 @@ interface SidebarMetricProps<T extends { difference: unknown }> {
    * Currently only behavior is doing that.
    */
   metricProperty?: string;
-  localeTextKey: keyof Locale;
+  localeTextKey: any;
```

Comment:
```suggestion
  localeTextKey: string;
```

nit; Would be a little more correct and at least you're enforcing a string on the public interface. Then the cast it to any (or better keyof) when looking up the value from the locale

Reply:
Rewrote this to `localeTextKey: keyof AllLanguages;`